src/test/java/org/zalando/fahrschein/PersistentCursorManagerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.zalando.fahrschein;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.zalando.fahrschein.domain.Cursor;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class PersistentCursorManagerTest {
+
+    public static final String CONSUMER = "any-conusmer";
+    public static final String EVENT_NAME = "any-event";
+    public static final String SCHEMA = "zc_foobar";
+
+    private JdbcTemplate jdbcTemplate;
+
+    @Before
+    public void setUp() throws Exception {
+        jdbcTemplate = mock(JdbcTemplate.class);
+    }
+
+    @Test
+    public void shouldExecuteSQLStatementForSuccessCallbackWithoutSchemaPrefix() throws Exception {
+
+        new PersistentCursorManager(jdbcTemplate, CONSUMER).onSuccess(EVENT_NAME, mock(Cursor.class));
```

Comment:
These tests are very tied to the implementation and will probably break on any refactoring of the `PersistentCursorManager`.

A better test could be similar to `PersistentCursorManagerIT` but with a modified search path of the database.

I would also be ok with merging this simple change without test.